src/scraper.py

```diff
@@ -9,8 +9,8 @@
 }
 
 URLS = [
-    "https://www.icij.org/investigations/pandora-papers/secret-real-estate-purchases-are-a-driving-force-behind-the-offshore-economy/",
     "https://www.icij.org/investigations/pandora-papers/former-czech-leaders-secret-french-estate-revealed-in-pandora-papers-listed-for-sale/",
+    "https://www.icij.org/investigations/panama-papers/20160404-azerbaijan-hidden-wealth/",
```

Comment:
rather than the overview article "secret real estate purchases", I cherry-pick a more focussed story on Azerbaijan, and the added benefit is that we get 1 article from pandora papers and 1 from panama papers